src/components/VSearchGrid.vue

```diff
@@ -44,7 +44,7 @@ export default {
       required: true,
     },
     searchType: {
-      type: String,
+      type: /** import('../store/types').SupportedSearchType */ String,
```

Comment:
The `@type` is still necessary as well as the `{}`. Additionally, because we're casting the `String` constructor to something specific, we need to wrap the thing we're casing in `()`.

Lastly, the PropType wrapper is necessary for Vue prop type annotations.

```suggestion
      type: /** @type {import('@nuxtjs/composition-api').PropType<import('../store/types').SupportedSearchType>} */ (String),
```

It's a little verbose :sweat_smile:. Once we can use proper TypeScript and can more easily import types the same way we import variables, these will be a little less cumbersome.